env/mock_env.cc

```diff
@@ -780,4 +780,14 @@ void MockEnv::FakeSleepForMicroseconds(int64_t micros) {
   fake_sleep_micros_.fetch_add(micros);
 }
 
+#ifndef ROCKSDB_LITE
+// This is to maintain the bahavior before swithcing from InMemoryEnv to MockEnv
+Env* NewMemEnv(Env* base_env) { return new MockEnv(base_env); }
```

Comment:
Agreed. There might be users legitimately using this function and MockEnv that is returned service all they needs. They do not need to learn that the implementation has changed.